binaries/cuprated/src/config.rs

```diff
@@ -98,43 +103,50 @@ config_struct! {
     #[derive(Debug, Deserialize, Serialize, PartialEq)]
     #[serde(deny_unknown_fields, default)]
     pub struct Config {
-        /// The network we should run on.
+        /// The network cuprated should run on.
         ///
-        /// Valid values: ["Mainnet", "Testnet" and "Stagenet"]
+        /// Valid values | "Mainnet", "Testnet", "Stagenet"
         pub network: Network,
 
         /// Enable/disable fast sync.
         ///
-        /// Fast sync skips verification of old blocks by comparing block hashes to a built-in hash file,
-        /// disabling this will significantly increase sync time. New blocks are still fully validated.
+        /// Fast sync skips verification of old blocks by
+        /// comparing block hashes to a built-in hash file,
+        /// disabling this will significantly increase sync time.
+        /// New blocks are still fully validated.
+        ///
+        /// Type         | boolean
+        /// Valid values | true, false
         pub fast_sync: bool,
 
         #[child = true]
-        /// The tracing/log output config.
+        /// Configuration for cuprated's logging system, tracing.
+        ///
+        /// Tracing is used for logging to stdout and files.
         pub tracing: TracingConfig,
 
         #[child = true]
-        /// The tokio config.
+        /// Configuration for cuprated's asynchronous runtime system, tokio.
         ///
-        /// Tokio is the async threadpool, used for network operations and the major service inside `cuprated`.
-
+        /// Tokio is used for network operations and the major service inside `cuprated`.
```

Comment:
small typo in my original PR

```suggestion
        /// Tokio is used for network operations and the major services inside `cuprated`.
```

Reply:
going to merge this now so we aren't waiting on it